docs/install/import.rst

```diff
@@ -2,6 +2,54 @@
 Import data
 ===========
 
+Import paths
+============
+
+.. danger::
+    With dynamic segmentation, importing paths is very risky if paths are already present in Geotrek,
```

Comment:
I would add "in the same area". Because importing some paths in an area without pre-existing ones, is not a problem. 
I also import paths all the time in areas that already have paths, I just take care in QGIS to msnap new paths with pre-existing ones. 
I use this method : https://makina-corpus.com/sig-webmapping/importer-une-couche-de-troncons-dans-geotrek

So I may wouldn't be so scary. 
But maybe with the UI import tool, it is more complicated and more risky. 

The good way to add paths in a Geotrek with existing one, is to prepare them in QGIS with displaying the path layer from Geotrek to snap new paths to existing ones, and to never import a path overriding an existing one.

Reply:
Ah yes good point, and I've forgottent this article from Mathieu :)

Reply:
Hello, I've made some improvements with your feedback